crates/iota-mainnet-unlocks/src/bin/generate_aggregated_data.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    collections::BTreeMap,
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+use anyhow::{Context, Result};
+use chrono::{DateTime, Utc};
+use csv::{Reader, Writer};
+use iota_mainnet_unlocks::store::{INPUT_FILE as OUTPUT_FILE, StillLockedEntry};
+use regex::Regex;
+use tempfile::{TempDir, tempdir};
+
+// Folders in the raw data repository that contain the CSV files.
+const FOLDERS: &[&str] = &[
+    "Assembly_IF_Members",
+    "Assembly_Investors",
+    "IOTA_Airdrop",
+    "IOTA_Foundation",
+    "New_Investors",
+    "TEA",
+    "Treasury_DAO",
+    "UAE",
+];
+
+/// Clones the repository containing raw data into a temporary directory.
+fn clone_repo(tmp_dir: &TempDir) -> Result<PathBuf> {
+    let repo_path = tmp_dir.path().join("new_supply");
+
+    let status = Command::new("git")
+        .args([
+            "clone",
+            "--depth",
+            "1",
+            "https://github.com/iotaledger/new_supply.git",
+        ])
+        .arg(&repo_path)
+        .status()
+        .context("failed to execute `git clone`")?;
+
+    if !status.success() {
+        anyhow::bail!("`git clone` failed with exit status: {}", status);
```

Comment:
There are ~10 more places like this
```suggestion
        anyhow::bail!("`git clone` failed with exit status: {status}");
```

Reply:
Ah yes, let me fix 👍

Reply:
Fixed in https://github.com/iotaledger/iota/pull/6123/commits/877c1d255c3ac56c7f5efe443f51f06a0054891d